docs/adr/0005-use-github-actions-for-vsp-platform-ci.md

```diff
@@ -44,24 +54,22 @@ what I’m saying is that if they love Circle and want to use it, we can support
 
 as long as they manage their own project-level credentials
 
-the problems with CircleCI for the Operations team is that we have many repos and managing those credentials at the repo level across all of those, as well as managing them all as separate pieces of code (when in reality they are 99% similar) is tedious, error prone, and makes Circle a bad fit (as long as the VA maintains those limitations)
-Github Actions is, I think, the best option.
+## Decision
 
- 
+Use GitHub Actions for VSP Platform CI.
```

Comment:
This whole thing seems to be more about 'why we don't like CircleCI'. It doesn't necessarily address the weaknesses of Github actions - including difficulty in debugging complicated workflows which we've seen in other projects at TW.

Reply:
Say more about this. For the most part our workflows will be exceedingly simple; i.e.: 
- Build a container or mirror a container from quay/dockerhub to ECR
- update a Kubernetes Manifest
- Make a release in Github

Interested to know what issues you have seen.

Reply:
Don't you plan to build a VSP/VA.gov release pipeline? Similar for other adjacent products?

I'll find some stories for you and share them here re: complex pipelines.

Reply:
We do not plan to use any of the ansible Build/Release/Deploy system that is currently in Jenkins. What would a VSP/VA.gov release pipeline look like in your mind?

Reply:
Keep in mind, we have settled on a CD product, with ArgoCD, we are not interested in complexity in the CI pipelines, they should _just_ be for testing, building containers, producing GH releases, and updating k8s manifests.

Reply:
> This whole thing seems to be more about 'why we don't like CircleCI'.

This is absolutely true, this was the impetus for making this decision. In fact, in my initial conversation, I said "I think the ADR for this would be an ADR to NOT use Circle for our own use, and the factors leading up to that, which is how I have been approaching it."

Reply:
"they should just be for testing, building containers, producing GH releases, and updating k8s manifests." - got it, understood, that makes sense. The testing part might be more or less complex esp if you want to have some way of ensuring certain tests are run across a broad variety of players - like security scans for example - but I take your point that you want this to just focus on CI.

Reply:
> I think the ADR for this would be an ADR to NOT use Circle for our own use…

This confusion is my doing, I honestly missed that distinction and mis-titled the reformatted ADR. I have since corrected it to, I hope, reflect Demian's intent better.